sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelper.java

```diff
@@ -86,12 +86,12 @@
     /**
      * Max number of properties.
      */
-    public static final int MAX_PROPERTY_COUNT = 5;
+    public static final int MAX_PROPERTY_COUNT = 20;
 
     /**
      * Max length of properties.
      */
-    public static final int MAX_PROPERTY_ITEM_LENGTH = 64;
+    public static final int MAX_PROPERTY_ITEM_LENGTH = 125;
```

Comment:
Why this specific number instead of something like 128?

Reply:
For feature parity with Xamarin Insights